examples/rccl/rccl-tests/src/common.h

```diff
@@ -20,8 +20,8 @@
 #include <fstream>
 #include <iostream>
 #include <pthread.h>
+#include <string.h>
 #include <string>
```

Comment:
```suggestion
#include <cstring>
```
If you just include <cstring>, does it still fix the issue in your environment? I think that that would be less confusing that two very similar "string" includes.